app/feature/travel-certificate/src/main/AndroidManifest.xml

```diff
@@ -1,2 +1,14 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest />
\ No newline at end of file
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+    <application>
+        <provider
+            android:name="com.hedvig.android.feature.travelcertificate.data.MyFileProvider"
+            android:authorities="com.hedvig.fileprovider"
+            android:exported="false"
+            android:grantUriPermissions="true">
+            <meta-data
+                android:name="android.support.FILE_PROVIDER_PATHS"
+                android:resource="@xml/file_paths" />
+        </provider>
+    </application>
+</manifest>
```

Comment:
We did have this https://github.com/HedvigInsurance/android/blob/67d1a4d8ce67926029372071d51e673470c65f3b/app/app/src/main/AndroidManifest.xml#L285-L293 for working with the camera pictures we get and store remporarily before uploading for the chat feature.
Could we maybe reuse that same one, or should we reuse that one so that we don't have two of them? Not sure about it.